.eslintrc.json

```diff
@@ -100,12 +100,13 @@
   "rules": {
     "@intlify/vue-i18n/valid-message-syntax": "warn",
     "@intlify/vue-i18n/no-duplicate-keys-in-locale": "warn",
-    "@intlify/vue-i18n/no-missing-keys-in-other-locales": "warn",
-    "@intlify/vue-i18n/no-dynamic-keys": "warn",
+    "@intlify/vue-i18n/no-raw-text": "off",
+    "@intlify/vue-i18n/no-dynamic-keys": "off",
     "@intlify/vue-i18n/no-unused-keys": [
       "warn",
       { "extensions": [".vue", ".ts"] }
-    ]
+    ],
+    "vue/no-v-html": "off"
```

Comment:
couldnt get this to work.. i dont think v-html is a necessary lint since we rarely show user input anyway